modules/op-installing-and-deploying-tekton-hub-on-an-openshift-cluster.adoc

```diff
@@ -19,7 +19,43 @@ If you are using Github Enterprise or Gitlab Enterprise, install and deploy {tek
 * Ensure that the {pipelines-title} Operator is installed in the default `openshift-pipelines` namespace on the cluster.
 
 [discrete]
-.Procedure
+.Procedure (Automatic)
+
+Starting with {pipelines-title} 1.8, you can automatically install and deploy {tekton-hub} with default configurations on a cluster by performing the following steps:
+
+. Create a `TektonHub` CR.
++
+[source,yaml]
```

Comment:
Could you please refer this [docs](https://github.com/tektoncd/operator/blob/main/docs/TektonHub.md#1-without-login-and-rating) once and change the yaml accordingly